interface/forms/dictation/common.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Dictation form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    cfapress <cfapress>
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Robert Down <robertdown@live.com>
+ * @copyright Copyright (c) 2008 cfapress <cfapress>
+ * @copyright Copyright (c) 2013-2017 bradymiller <bradymiller@users.sourceforge.net>
+ * @copyright Copyright (c) 2017-2023 Robert Down <robertdown@live.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ **/
+
+require_once(__DIR__ . "/../../globals.php");
+require_once("$srcdir/api.inc.php");
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Core\Header;
+
+$returnurl = 'encounter_top.php';
+$mode = filter_input(INPUT_GET, 'mode', FILTER_SANITIZE_SPECIAL_CHARS);
+$id = filter_input(INPUT_GET, 'id', FILTER_SANITIZE_NUMBER_INT);
+
+if ($mode == 'new') {
+    $save_action = "save.php?mode=new";
+} else {
+    $save_action = "save.php?mode=update&id=" . $id;
+}
+?>
+<html>
+<head>
+    <title><?php echo xlt("Dictation"); ?></title>
+    <?php Header::setupHeader(['ckeditor', 'angular', 'angular-sanitize']);?>
+</head>
+<body>
+<div class="container mt-3">
+    <div class="row">
+        <div class="col-12">
+            <h2><?php echo xlt("Dictation"); ?></h2>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-12" id="toptext">
+            <form name="my_form" method=post action="<?php echo $rootdir;?>/forms/dictation/"<?php echo $save_action ?> onsubmit="return top.restoreSession()">
+                <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                <fieldset>
+                    <legend><?php echo xlt('Dictation')?></legend>
+                    <div class="container">
+                        <div class="form-group">
+                            <textarea name="dictation" id="editor1" class="form-control" cols="80" rows="15"></textarea>
```

Comment:
don't these need to be populated when updating/viewing a form?

Reply:
also, for all the forms, can double click in the text field and it will open up the template editor. i think this is using a ckeditor although @sjpadgett is best one to know this since he added that feature

Reply:
@bradymiller that feature only seems to work on a plain textbox or text area. The ckeditor replaces the textarea so the feature that @sjpadgett has injected in the form page does not work. It still works on the additional info textarea.

Reply:
I didn't implement ckeditor. It was originally put in for nation notes! I merely co-opted to get rid of my original editor for portal something I now wished I hadn't done!
This might be a good opportunity to try out another editor. SN thought maybe TinyMCE might be an option.
Also ckeditor does not have to be in a textbox any container will do.

Reply:
I looked at the TinyMCE. https://www.tiny.cloud/docs/tinymce/latest/php-projects/
Project admins are going to have to bring this in. I could not find a WYSIWYG named SN. 
TinyMCE requires an API key.

Reply:
SN is stephen nielson lol sorry
so much for open source freedom!